modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/ServerImpl.java

```diff
@@ -7364,11 +7362,11 @@ else if (msg instanceof TcpDiscoveryRingLatencyCheckMessage) {
                     U.interrupt(clientMsgWrk.runner());
                 }
 
-                U.closeQuiet(sock);
+                U.close(sock, log);
```

Comment:
`closeQuiet` has an overload for `Socket` which performs additional steps. Please create a similar overload for `close`, otherwise we break [IGNITE-11298](https://issues.apache.org/jira/browse/IGNITE-11298) changes.

Reply:
@ptupitsyn fixed.